configs/bifrost.yml

```diff
@@ -0,0 +1,13 @@
+endpoint: wss://bifrost-parachain.api.onfinality.io/public-ws
+mock-signature-host: true
+block: ${env.BIFROST_BLOCK_NUMBER}
+db: ./db.sqlite
+
+import-storage:
+  System:
+    Account:
+      -
+        -
+          - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+        - data:
```

Comment:
@goncer please set providers to 1 so you don't have to updated this later

Reply:
what do you mean by providers? not sure I understand you :upside_down_face:

Reply:
updated :crossed_fingers:

Reply:
yes, latest substrate version requires providers so when you upgrade it will still work